bee-ledger/src/workers/pruning/batch.rs

```diff
@@ -42,8 +41,8 @@ pub struct Edge {
     pub to_child: MessageId,
 }
 
-pub fn prune_confirmed_data<S: StorageBackend>(
-    tangle: &Tangle<S>,
+pub fn batch_prunable_confirmed_data<S: StorageBackend>(
```

Comment:
nitpick: Would `prunable_confirmed_data_batch` make it more intuitive?

Reply:
"batch_" is used as a verb here, so in terms of "to batch sth".